src/Facebook/GraphNodes/GraphCoverPhoto.php

```diff
@@ -0,0 +1,82 @@
+<?php
+/**
+ * Copyright 2014 Facebook, Inc.
+ *
+ * You are hereby granted a non-exclusive, worldwide, royalty-free license to
+ * use, copy, modify, and distribute this software in source code or binary
+ * form for use in connection with the web services and APIs provided by
+ * Facebook.
+ *
+ * As with any software that integrates with the Facebook platform, your use
+ * of this software is subject to the Facebook Developer Principles and
+ * Policies [http://developers.facebook.com/policy/]. This copyright notice
+ * shall be included in all copies or substantial portions of the software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+ * THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+ * DEALINGS IN THE SOFTWARE.
+ *
+ */
+namespace Facebook\GraphNodes;
+
+/**
+ * Class GraphCoverPhoto
+ *
+ * @package Facebook
+ */
+class GraphCoverPhoto extends GraphObject
+{
+	/**
+     * Returns the id of cover if it exists
+     *
+     * @return int|null
+     */
+    public function getId()
+    {
+        return $this->getProperty('id');
+    }
+	
+	/**
+     * Returns the cover_id of cover if it exists
+     *
+     * @return int|null
+     */
+    public function getCoverId()
+    {
+        return $this->getProperty('cover_id');
```

Comment:
As Shireesh Asthana mention, it seem to be backward compatibility issue!

Must I return the support of `cover_id` again? because I remove it from the last push